app/src/main/java/org/systers/mentorship/view/activities/SignUpActivity.kt

```diff
@@ -122,4 +126,9 @@ class SignUpActivity : BaseActivity() {
         startActivity(intent)
         finish()
     }
+
+    override fun onSupportNavigateUp(): Boolean {
+        onBackPressed()
```

Comment:
Any reason you are calling `onBackPressed()` instead of the usual `finish()`?

Reply:
TBH I've no idea. 👀